requirements.txt

```diff
@@ -0,0 +1,2 @@
+molecule==2.22
```

Comment:
Your tests are working with Molecule 2.x, changes required for 3.x are pretty hefty. So this is why I decided to pin it (for now).